index.js

```diff
@@ -10,6 +10,8 @@ module.exports = {
 	rules: {
 		'arrow-body-style': ['off'],
 		'consistent-return': ['off'],
+		'function-paren-newline': 'off',
```

Comment:
i prefer the `function-paren-newline: ["error", "consistent"]` [option](https://eslint.org/docs/rules/function-paren-newline#:~:text=Examples%20of%20correct%20code%20for%20this%20rule%20with%20the%20%22consistent%22%20option%3A)

Reply:
(ie. if one param is on a newline, all should be on newlines)